ogr/utils.py

```diff
@@ -164,7 +166,7 @@ def __str__(self) -> str:
             f"RequestResponse("
             f"status_code={self.status_code}, "
             f"ok={self.ok}, "
-            f"content={self.content}, "
+            f"content={self.content.decode()}, "
```

Comment:
@jscotka is this ok? (I hope we don't break anything related to requre.)

Reply:
should be okay requre should be trasparent.

Reply:
jscotka said it should be OK..